keps/sig-network/3015-prefer-same-node/README.md

```diff
@@ -495,6 +496,7 @@ N/A
 - Initial proposal as "Node-level topology": 2022-01-15
 - Initial proposal as `TrafficDistribution: PreferSameNode`: 2025-02-06
 - Added `TrafficDistribution: PreferSameZone`: 2025-02-08
+- Implemented as Alpha in k8s 1.33: 2025-05-30
```

Comment:
Also for question

"Will enabling / using this feature result in increasing size or count of the existing API objects"

the answer should include that the ForNode hints can be added.